avocado/utils/memory.py

```diff
@@ -141,7 +141,7 @@ def memtotal_sys():
     block_size = int(open(os.path.join(sys_mempath,
                                        'block_size_bytes'),
                           "r").read().strip(), 16)
-    return (no_memblocks * block_size)/1024.0
+    return (no_memblocks * block_size) / 1024.0
```

Comment:
Please split those style fixes into a separate commit, it decreases the readability of the change and again breaks reverts as in case of revert this style fix would be gone as well.